foundation/foundation-configuration-store/pom.xml

```diff
@@ -60,8 +60,8 @@
                     </systemPropertyVariables>
                     <!-- set environment variables for unit test -->
                     <environmentVariables>
-                        <BASE_URI>/tmp/configurations/base</BASE_URI>
-                        <ENVIRONMENT_URI>/tmp/configurations/example-env</ENVIRONMENT_URI>
+                        <BASE_PROPERTY_URI>${project.basedir}/src/test/resources/configurations/base</BASE_PROPERTY_URI>
+                        <ENVIRONMENT_PROPERTY_URI>${project.basedir}/src/test/resources/configurations/example-env</ENVIRONMENT_PROPERTY_URI>
```

Comment:
I: Moved unit tests to use existing policies within `/src/test/resources/` rather than writing them to a `/tmp` directory outside of the repo